pages/faq.html

```diff
@@ -268,12 +268,12 @@ <h2 id="instructors">Instructor Training</h2>
   <dt id="how-become-instructor">How can I become an instructor myself?</dt>
   <dd>
     <p>
-      We offer a <a href="{{site.training_url}}">free online training course</a>
+      We offer a <a href="{{site.baseurl}}/pages/lessons.html">free online training course</a>
       for people who would like to teach Software Carpentry,
       and to learn more about teaching in general along the way.
       The course takes 2-4 hours a week for 12-14 weeks (depending on interruptions);
       most work is done on trainees' own time,
```

Comment:
should end the sentence here - we no longer have a separate blog for the training course.